src/ActionModal/components/ActionModal.vue

```diff
@@ -47,7 +47,8 @@
       </template>
       <TmDataLoading
         v-else-if="
-          $apollo.queries.overview.loading || $apollo.queries.balances.loading
+          $apollo.loading &&
```

Comment:
whyyyy? still loading there

Reply:
hmm maybe your right, need to try it out. as we load some stuff on every step, right?

Reply:
I can confirm this works, but you're free to try it out. Like you said, `$apollo.loading` is true if any of the three queries (balances, overview and gasEstimate) is still loading

Reply:
this is not a good strategy from a ux perspective. the step should always load instantly. if there is data that is being loaded we should show a loader inline with the data and restrict the user from going forward. blocking this components like this is no bueno.

Reply:
Sounds cool, I like the idea. So basically what you are proposing is loading as you go, show what you have at each point, right?